File: cpp-programs/functions/factorial.cpp
#include<string.h>
#include<bits/stdc++.h>
using namespace std;

long  fact(long);

int main()
{

long n,f;

cout<<"Enter the number : ";
cin>>n;

f = fact(n);
cout<<"factorial = "<<f<<endl;

}

long fact(long a)
{
 long f = 1;
 int i ;
 
 for(i=1;i<=a;i++)
 {
  f = f * i;
 }
 return f;
}

File: practice-code-orange/programme-c#/factorial.cs
using System;
namespace Fact
{

class Factorial
{
 int factorial(int num)
 {
    if(num==1)
    {
      return 1;
    }
    else
    {
      return num * factorial(num-1);
    }
  
 
  }
 
 static void Main(string[] args)
 {
  Factorial obj= new Factorial();
  int value = obj.factorial(6);
  Console.WriteLine("Factorial of 6 : {0}",value);
 }
} 
}

